cpp/src/barretenberg/proof_system/circuit_constructors/standard_circuit_constructor.hpp

```diff
@@ -44,7 +53,7 @@ class StandardCircuitConstructor : public CircuitConstructorBase<STANDARD_WIDTH>
     StandardCircuitConstructor(const StandardCircuitConstructor& other) = delete;
     StandardCircuitConstructor(StandardCircuitConstructor&& other) = default;
     StandardCircuitConstructor& operator=(const StandardCircuitConstructor& other) = delete;
-    StandardCircuitConstructor& operator=(StandardCircuitConstructor&& other) = default;
```

Comment:
This is not used anywhere, and it became implicitly deleted with the change that defined selector aliases using `std::get`, so I removed it in favor of the clearer syntax. I see no reason to have a move assignment operator of this type.

Reply:
It may be better to explicitly `delete` this rather than removing it altogether

Reply:
Good call